ccd-definition/ComplexTypes/DQ/EmploymentTypeComplexTypeDQ.json

```diff
@@ -0,0 +1,11 @@
+[
```

Comment:
Wouldn't be enough to use a MultiSelectList field instead of a complex field that only contains the MultiSelectList?

Reply:
removed this complex type as suggested